flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkConfigManager.java

```diff
@@ -87,8 +89,15 @@ public Configuration load(Key k) {
                                 });
 
         updateDefaultConfig(defaultConfig);
+        ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
+        executorService.scheduleWithFixedDelay(
```

Comment:
Calling the `Cache#cleanUp()` periodically and cleaning up the tmp files in the `removalListener` could fix the issue in FLINK-27303. But actually the pod template, config, log tmp files are useless after the successful submission.

Do you think we could clean up the tmp files more aggressive? Imagine the operator is managing thousand of jobs, the current implementation will still have so many tmp files.

Reply:
You are right. We could not simply delete the tmp files in case of roll-back.